hudi-client/hudi-client-common/src/main/java/org/apache/hudi/table/marker/DirectWriteMarkers.java

```diff
@@ -189,13 +190,12 @@ private Option<Path> create(Path markerPath, boolean checkIfExists) {
     } catch (IOException e) {
       throw new HoodieIOException("Failed to make dir " + dirPath, e);
     }
-    try {
+    try (FSDataOutputStream fsDataOutputStream = fs.create(markerPath, false)) {
       if (checkIfExists && fs.exists(markerPath)) {
         LOG.warn("Marker Path=" + markerPath + " already exists, cancel creation");
```

Comment:
The logic is not equivalent with before.

Reply:
ok, i restore it